backup/moodle2/restore_opencast_block_task.class.php

```diff
@@ -103,12 +119,13 @@ protected function define_my_steps() {
         }
 
         foreach ($ocinstances as $ocinstance) {
+            $settingname = 'opencast_videos_' . $ocinstance->id . '_included';
             // Settings, does not exists, if opencast system does not support copying workflow.
-            if (!$this->setting_exists('opencast_videos_include_' . $ocinstance->id)) {
+            if (!$this->setting_exists($settingname)) {
                 continue;
             }
 
-            if (!$this->get_setting_value('opencast_videos_include_' . $ocinstance->id) &&
+            if (!$this->get_setting_value($settingname) &&
```

Comment:
This is I believe, in order to escape the steps when other modes of course reuse is performed eg. restore etc and that might happen when a restore is using an old backup file and there is no such step defined, therefore, the restore process does not throw error.
And this at the same time makes sure that if you mode import the step has to be there, because import is a sequence of backup then import!